hax/hax/handler.py

```diff
@@ -53,9 +53,11 @@ def __init__(self, q: Queue, motr: Motr, herald: DeliveryHerald,
                          name='qconsumer',
                          args=(q, motr))
         self.is_stopped = False
+        self.is_finalized = False
```

Comment:
I can't see where this field is used. Is it really needed?

Reply:
Removed.